modules/abstract-lightning/src/wallet/lightning.ts

```diff
@@ -104,7 +104,7 @@ export class SelfCustodialLightningWallet implements ILightningWallet {
   async createInvoice(params: CreateInvoiceBody): Promise<Invoice> {
     const createInvoiceResponse = await this.wallet.bitgo
       .post(this.wallet.baseCoin.url(`/wallet/${this.wallet.id()}/lightning/invoice`))
-      .send(CreateInvoiceBody.encode(params))
+      .send(t.exact(CreateInvoiceBody).encode(params))
```

Comment:
this should have been a different commit but with this we won't send any extra params to WP if client accidentally passes params not in the codec.